### DISCUSSION TITLE
Issuing "command 'extension.js-debug.setAutoAttachVariables' not found" when start VSCode

### DISCUSSION THREAD

**Discussion by kylefelipe** [community/community#42706] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

Hi, few days ago my copilot starts to show the following message when open VSCode

GitHub Copilot could not connect to server. Extension activation failed: "command 'extension.js-debug.setAutoAttachVariables' not found"

S.O.: Linux UBUNTU 
VSCode Version: 1.74.2
Electron: 19.1.8
Node.js: 16.14.2
V8: 10.2.154.15-electron.0

**Comment by Akash1134** [comment]
Hi @kylefelipe Sorry that you cannot use Copilot with VSCode. An out-of-date IDE could be causing this issue. 
- Make sure your IDE is up to date
- If you're not correctly signed in to GitHub CoPilot - check that you're signed in to GitHub CoPilot.
E.g. In VSCode look on the bottom left, if you see this, you might not be signed in.
- Try signing out of the IDE and then reopen your IDE to see if that works.

**Reply by kylefelipe** [reply]
My IDE is up to date.. (1.74.2)
im sure that im sined in (i can sync account with gh)
I signed out and repened my IDE, and still showing the message...

**Comment by parifuture** [comment, marked as answer]
hey @kylefelipe can you in your vscode search for all the disabled extensions and see if "Javascript debugger" is disabled, I was having this issue for last 2 weeks and accidentally stumbled on it. Enabling and re-starting vscode fixed the issue. 
![Uploading Screenshot 2023-01-14 at 1.36.30 AM.png…]()

**Reply by kylefelipe** [reply]
Thank you so much for your reply... just solved my issue here...
